editor/editor/mode/visual.js

```diff
@@ -19,13 +19,25 @@ function Blocks( { blocks, onChange } ) {
 
 	return (
 		<div className="editor-mode-visual">
-			<div>
-				{ blocks.map( ( block, index ) =>
-					<div key={ index }>
-						{ wp.blocks.getBlockSettings( block.blockType ).edit( block.attributes, onChangeBlock( index ) ) }
-					</div>
-				) }
-			</div>
+			{ blocks.map( ( block, index ) => {
+				const settings = wp.blocks.getBlockSettings( block.blockType );
+
+				let BlockEdit;
+				if ( settings ) {
```

Comment:
>I think we can assume this is always defined, right!

I guess this depends if the editor implementation should trust itself to know that elsewhere it has assigned the unknown type handler. 😄  I might be erring too far on the side of safety here. Would certainly be more convenient to be able to assume that `settings` is always truthy.